.github/ISSUE_TEMPLATE/10_generated_package.md

```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```

Comment:
Merge conflict unresolved?

Reply:
Thanks for spotting. It was a big merge conflict, so I missed this one.